PointOfInteraction/SmartSpot/doc/spec.md

```diff
@@ -0,0 +1,70 @@
+# SmartSpot
+
+## Description
+
+Smart Spots are devices which provide the the technology which allows users to get access to smart points of interaction so that they can obtain extra information (infotainment, …), provide suggestions (suggestions mailbox, …) or generate new content (co-creation, …). The data model contains resources to configure the interaction service such as the broadcasted URL (typically shortened), the interval between broadcasts (frequency), the availability of the service, transmission power depending on the area to be covered, etc.
+
+In addition to the presented data model, this entity inherits the Device data model. This means that by hierarchy, the `SmartSpot` entity type is a subtype of `[Device](https://github.com/Fiware/dataModels/blob/master/Device/Device/doc/spec.md)` and as a result it can be the subject of any of the properties that an entity of type `[Device](https://github.com/Fiware/dataModels/blob/master/Device/Device/doc/spec.md)` may have.
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `SmartSpot`.
+
++ `announcedUrl` : URL broadcasted by the device.
+    + Attribute type: [URL](https://schema.org/URL)
+    + Mandatory    
+
++ `signalStrenght` : Signal strength to adjust the announcement range.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Allowed values: "lowest", "medium" or "highest". 
+    + Mandatory    
+
++ `bluetoothChannel` : Bluetooth channels where to transmit the announcement.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Allowed values: "37", "38", "39", "37,38", "38,39", "37,39" or "37,38,39".
+    + Mandatory  
+
++ `areaCovered` : Radius of the spot coverage area in meters.
```

Comment:
Unless this is inherited from any other specification, I would not recommend using the same property names with distinct semantics. There is another `areaCovered` property at the `SmartPointOfInterest` level refers to a real geometric area. I would recommend renaming this property to something such as `areaCoveredRadius` or alike :)